testsuite/src/it/java/org/graalvm/tests/integration/utils/GDBSession.java

```diff
@@ -134,6 +134,52 @@ public CP[] get(boolean inContainer) {
                                 Pattern.compile(".*ClassA\\(int myNumber, String myString\\).*",
                                         Pattern.DOTALL)),
                 };
+            } else if (UsedVersion.jdkFeature(inContainer) == 21) {
+                return new CP[]{
+                        SHOW_VERSION,
+                        new CP("run < ./test_data_small.txt\n",
+                                Pattern.compile(".*fdc7c50f390c145bc58a0bedbe5e6d2e35177ac73d12e2b23df149ce496a5572.*exited normally.*",
+                                        Pattern.DOTALL)),
+                        new CP("info functions .*smoke.*\n",
+                                Pattern.compile(
+                                        ".*File debug_symbols_smoke/ClassA.java:.*" +
+                                                "java.lang.String \\*debug_symbols_smoke.ClassA::toString\\(\\).*" +
+                                                "File debug_symbols_smoke/Main.java:.*" +
+                                                "void debug_symbols_smoke.Main\\$\\$Lambda/0x.*::accept\\(java.lang.Object\\*\\).*" +
```

Comment:
This is the line that got changed. Lambdas are named differently in JDK 21.

Reply:
You mean in order to match both JDK < 21 and JDK >= 21 lambda names. Sure it would.

Reply:
Yes, that's the idea.